src/main/java/org/jabref/logic/integrity/TitleChecker.java

```diff
@@ -48,13 +48,18 @@ public TitleChecker(BibDatabaseContext databaseContext) {
             valueOnlySpacesWithinCurlyBraces = matcher.replaceAll("");
         }
 
-        boolean hasCapitalLettersThatBibtexWillConvertToSmallerOnes = HAS_CAPITAL_LETTERS
-                .test(valueOnlySpacesWithinCurlyBraces);
-
-        if (hasCapitalLettersThatBibtexWillConvertToSmallerOnes) {
-            return Optional.of(Localization.lang("capital letters are not masked using curly brackets {}"));
+        // Delimiters are . ! ? ; :
+        String[] delimitedStr = valueOnlySpacesWithinCurlyBraces.split("(\\.|\\!|\\?|\\;|\\:)");
+        for (String subStr : delimitedStr) {
+            subStr = subStr.trim();
+            if (subStr.length() > 0) {
+                subStr = subStr.substring(1);
+            }
+            boolean hasCapitalLettersThatBibtexWillConvertToSmallerOnes = HAS_CAPITAL_LETTERS.test(subStr);
+            if (hasCapitalLettersThatBibtexWillConvertToSmallerOnes) {
```

Comment:
I believe, as `hasCapitalLettersThatBibtexWillConvertToSmallerOnes` is not reused, it can be inlined in the `if` predicate and still be meaningful. 😉

Reply:
I also changed this :)